src/components/Drawer.js

```diff
@@ -0,0 +1,62 @@
+const React = require('react');
+const Velocity = require('velocity-animate');
+
+const Button = require('../components/Button');
+
+const Drawer = React.createClass({
+  propTypes: {
+    onClose: React.PropTypes.func.isRequired
+  },
+
+  componentDidMount () {
+    const el = this._container;
+    const transition = { right: 0 };
+    const options = {
+      duration: 500
+    };
+
+    Velocity(el, transition, options);
```

Comment:
Yeah. I had issues with that as well. I messed around with an internal state variable in the child component but didn't come up with any solution that I was pleased with.